README.md

```diff
@@ -1,2 +1,40 @@
-# terraform-provider-flux
-Terraform provider for Flux v2
+# Terraform provider FluxV2
+
+This is a Terraform provider for Flux v2, it enables bootstrap a Kubernetes custer with Flux v2 using terraform.
+
+## Example Usage
+
+The provider is consists of two data sources `flux_install` and `flux_sync` the data sources are corresponding to [fluxv2 manifests](https://pkg.go.dev/github.com/fluxcd/flux2@v0.2.1/pkg/manifestgen)
+
+The data sources are returing `YAML` manifest so a second provider is needed to apply the manifest into the Kubernetes cluster. See example folder.
+
+The `flux_install` generates manifests to install the `fluxv2` components.
+
+```hcl
+# Flux
+data "flux_install" "main" {
+  target_path = "fleet/infra"
```

Comment:
The `target_path` should match the CLI example = `target_path = "staging-cluster"`

Reply:
fixed